src/features/bounds/model.ts

```diff
@@ -96,6 +98,50 @@ export function getAbsoluteBounds(element: SModelElement): Bounds {
     }
 }
 
+/**
+ * Returns the "client-absolute" bounds of the specified `element`.
+ *
+ * The client-absolute bounds are relative to the entire browser page.
+ *
+ * @param element The element to get the bounds for.
+ * @param domHelper The dom helper to obtain the SVG element's id.
+ * @param viewerOptions The viewer options to obtain sprotty's container div id.
+ */
+export function getAbsoluteClientBounds(element: SModelElement, domHelper: DOMHelper, viewerOptions: ViewerOptions): Bounds {
+    let x = 0;
+    let y = 0;
+    let width = 0;
+    let height = 0;
+
+    const svgElementId = domHelper.createUniqueDOMElementId(element);
+    const svgElement = document.getElementById(svgElementId);
+    if (svgElement) {
+        const rect = svgElement.getBoundingClientRect();
+        const body = document.body;
+        const docElement = document.documentElement;
+        const scrollTop = window.pageYOffset || docElement.scrollTop || body.scrollTop;
+        const scrollLeft = window.pageXOffset || docElement.scrollLeft || body.scrollLeft;
```

Comment:
Why not use `window.scrollX` and `window.scrollY` as [documented on MDN](https://developer.mozilla.org/en-US/docs/Web/API/Element/getBoundingClientRect)? Maybe then you wouldn't need to take into account the `clientTop` / `clientLeft` values.

Reply:
You're right, this seems to work too. Thanks!